package.json

```diff
@@ -14,7 +14,8 @@
     "node": ">=8"
   },
   "scripts": {
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "e2e:defaults": "start-server-and-test 'node ./scripts/createAppAndStartServer foo --acceptDefaults' 3001 'cypress run --spec cypress/integration/createBisonAppWithDefaults.test.js'"
```

Comment:
since we're testing this via the cli, a single e2e test for each command line seemed like the best route to go. I originally tried doing this using cypress commands and setup/teardown was very painful.

This is pretty verbose, but it's also tests the app like a user would

Reply:
What do you think if we make this `test:e2e`? I could see us writing different tests (unit, integration, etc.) to test individual parts of this tool independently.